deploy/api.yaml

```diff
@@ -82,8 +82,132 @@ paths:
         passthroughBehavior: "when_no_match"
         contentHandling: "CONVERT_TO_TEXT"
         type: "aws_proxy"
+  /token:
+    parameters:
+      - $ref: "#/components/parameters/SessionHeader"
+      - $ref: "#/components/parameters/GrantType"
+      - $ref: "#/components/parameters/Code"
+      - $ref: "#/components/parameters/RedirectUri"
+      - $ref: "#/components/parameters/ClientId"
+    post:
+      requestBody:
+        content:
+          application/x-www-form-urlencoded:
+            schema:
+              $ref: "#/components/schemas/TokenRequest"
+        required: true
+      responses:
+        "201":
+          description: "201 response"
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/TokenResponse"
+        "400":
+          description: "400 response"
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Error"
+        "500":
+          description: "500 response"
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Error"
+
+      x-amazon-apigateway-request-validator: " Validate token request"
+      x-amazon-apigateway-integration:
+        httpMethod: "POST"
+        uri:
+          Fn::Sub: arn:aws:apigateway:${AWS::Region}:lambda:path/2015-03-31/functions/${AccessTokenFunction.Arn}/invocations
+        responses:
+          default:
+            statusCode: "200"
+        passthroughBehavior: "when_no_match"
+        contentHandling: "CONVERT_TO_TEXT"
+        type: "aws_proxy"
 components:
+  parameters:
+    SessionHeader:
+      name: session_id
```

Comment:
IPV Core won't be sending the `session_id` in this OAuth flow call. Rather, we use the authorization `code` to look up the session id